src/D2L.CodeStyle.Analyzers/Language/RequireNamedArgumentsAnalyzer.cs

```diff
@@ -209,19 +248,20 @@ ArgumentListSyntax args
 					paramName = param.Name;
 				}
 
-				string text = param.OriginalDefinition.Type.ToMinimalDisplayString( model, 0 );
-
-				string psuedoName = GetPsuedoName( arg );
-
-				if( psuedoName != null ) {
-					bool matchesParamName = string.Equals(
-						psuedoName,
-						param.Name,
-						StringComparison.OrdinalIgnoreCase
-					);
-
-					if( matchesParamName ) {
-						continue;
+				if( !requireNamedArguments ) {

```

Comment:
Skip the psuedo pass if named arguments are required

Reply:
Hmmm, why though?